packit/status.py

```diff
@@ -50,23 +49,24 @@ def __init__(
         self.up = upstream
         self.dg = distgit
 
-    def get_downstream_prs(self, number_of_prs: int = 5) -> None:
+    def get_downstream_prs(self, number_of_prs: int = 5) -> List:
         """
         Get specific number of latest downstream PRs
         :param number_of_prs: int
-        :return: None
+        :return: List of downstream PRs
         """
+        table = None
```

Comment:
Since we have a list as a return value (`-> List`), we should have an empty string in case of no PRs.

```suggestion
        table = []
```

Reply:
I will use `List[Tuple[int, str, str]] = []`